package.json

```diff
@@ -10,7 +10,8 @@
     "start:js": "webpack --progress --watch",
     "start": "npm run start:js & bundle exec jekyll serve --livereload",
     "test:a11y-desktop": "pa11y-ci --config ./.pa11yci-desktop --sitemap http://127.0.0.1:4000/sitemap.xml",
-    "test:a11y-mobile": "pa11y-ci --config ./.pa11yci-mobile --sitemap http://127.0.0.1:4000/sitemap.xml"
+    "test:a11y-mobile": "pa11y-ci --config ./.pa11yci-mobile --sitemap http://127.0.0.1:4000/sitemap.xml",
+    "test:ui": "npm run start:js & bundle exec jekyll serve --detach && playwright test"
```

Comment:
I ran `npm run test:ui` then received output that indicated I needed to run `npx playwright install`.  All 23 tests failed with a message that mentioned `npx playwright install`. See below.

```
Running 23 tests using 2 workers
  1) [chromium] › search-test-desktop.spec.js:3:1 › Desktop tests ==================================

    browserType.launch: Executable doesn't exist at /Users/eileenmcfarland/Library/Caches/ms-playwright/chromium-978106/chrome-mac/Chromium.app/Contents/MacOS/Chromium
    ╔═════════════════════════════════════════════════════════════════════════╗
    ║ Looks like Playwright Test or Playwright was just installed or updated. ║
    ║ Please run the following command to download new browsers:              ║
    ║                                                                         ║
    ║     npx playwright install                                              ║
    ║                                                                         ║
    ║ <3 Playwright Team                                                      ║
    ╚═════════════════════════════════════════════════════════════════════════╝

```

I tried adding `npx playwright install` and `npm playwright install` to the script, however, it didn't fix the problem.

Reply:
Did you run `npm install` when you pulled down this branch?

Reply:
Update: all the tests now pass!  Yay!

Reply:
Sweet!